substrafl/dependency/path_management.py

```diff
@@ -0,0 +1,264 @@
+import logging
+import shutil
+from abc import ABC
+from abc import abstractmethod
+from pathlib import Path
+from typing import List
+from typing import Optional
+from typing import Set
+
+logger = logging.getLogger(__name__)
+
+TMP_SUBSTRAFL_PREFIX = "tmp_substrafl"
+# Common paths regex we want to exclude, partially based on
+# https://github.com/github/gitignore/blob/main/Python.gitignore
+EXCLUDED_PATHS_REGEX_DEFAULT = [
```

Comment:
I'm wondering if the definition of the default constants is not long enough to justify moving it to its own file 🤷

Reply:
That sounds like a good idea, but how would you name it?

Reply:
`default.py`? `constants.py`?

Reply:
renamed to `constants.py`